core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.directive.html

```diff
@@ -65,13 +65,13 @@ <h3 class="skill-card-header-text">Details</h3>
               <a ng-click="changeActiveWorkedExampleIndex($index)"
                  class="oppia-rule-tab protractor-test-worked-example-<[$index]>"
                  ng-class="{'oppia-rule-tab-active': activeWorkedExampleIndex === $index}">
-                <summary-list-header index="$index"
-                                     disable-sorting="bindableFieldsDict.displayedWorkedExamples.length === 1"
-                                     summary="getWorkedExampleSummary(workedExample.getQuestion().html)"
-                                     short-summary="getWorkedExampleSummary(workedExample.getQuestion().html)"
-                                     is-active="$index === activeWorkedExampleIndex"
-                                     is-delete-available="isEditable"
-                                     on-delete-fn="deleteWorkedExample">
+                <summary-list-header [index]="$index"
```

Comment:
```suggestion
                <oppia-summary-list-header [index]="$index"
```

Lets rename it everywhere.

Reply:
done